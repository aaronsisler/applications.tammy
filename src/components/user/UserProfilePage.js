import React from 'react';
import UserProfileContainer from './UserProfileContainer';

export default class UserProfilePage extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div id="user_profile_page">
                <UserProfileContainer />
            </div>
        );
    }
}
