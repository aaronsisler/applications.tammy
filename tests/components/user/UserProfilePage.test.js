import React from 'react';
import { shallow } from 'enzyme';
import UserProfilePage from '../../../src/components/user/UserProfilePage';

describe('UserProfilePage', () => {
    it('should render UserProfilePage correctly', () => {
        const wrapper = shallow(<UserProfilePage />)
        expect(wrapper).toMatchSnapshot();
    })
})
