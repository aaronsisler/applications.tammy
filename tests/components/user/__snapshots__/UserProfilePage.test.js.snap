// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfilePage should render UserProfilePage correctly 1`] = `
<div
  id="user_profile_page"
>
  <Connect(UserProfileContainer) />
</div>
`;
